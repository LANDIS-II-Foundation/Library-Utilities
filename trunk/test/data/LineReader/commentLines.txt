#  This file just has comment lines in it.
		# Comment line with leading whitespace.
#################
