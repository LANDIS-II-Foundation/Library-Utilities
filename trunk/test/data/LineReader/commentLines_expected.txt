1:#  This file just has comment lines in it.
2:		# Comment line with leading whitespace.
3:#################
